### DISCUSSION TITLE
Is scrobbling needed after using the stream endpoint?

### DISCUSSION THREAD

**Discussion by khers** [navidrome/navidrome#2804] [Q&A] [answered]
I am working on an Open Subsonic client and I am trying to get listen reporting working. When I use the stream endpoint, is that counted as a listen implicitly or do I still need to scrobble the listen?

**Comment by deluan** [comment, marked as answer]
Yes, a call to `/scrobble` is required and it is the only guaranteed way for OpenSubsonic servers to register a listen. From [OpenSubsonic docs](https://opensubsonic.netlify.app/docs/endpoints/stream/#result):

> OpenSubsonic servers must not count access to this endpoint as a play and increase playcount. Clients can use the [Scrobble](https://opensubsonic.netlify.app/docs/endpoints/scrobble) endpoint to indicate that a media is played ensuring proper data in all cases.

Here is some background on this decision: https://github.com/navidrome/navidrome/issues/593#issuecomment-716002287
